auth/authRoute.js

```diff
@@ -119,7 +121,7 @@ router.post("/register", (req, res) => {
       .then(user => {
         const token = generateToken(user);
         res.status(201).json({
```

Comment:
We could `delete user.password` and then send back the user object without that key value pair.

Reply:
I've already got that set up in my PR. No worries to make any changes here @BDesselle